internal/cache/base.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 var (
-	cacheDuration     = 1 * time.Minute
```

Comment:
I kind of preferred the old function to expose it but it doesn't matter much as this is about naming. I think we ought to remove global variables to some extent in the future honestly. I also don't know in which capacity the `FYNE_CACHE` env var is used. Removing it would allow us to have these as constants instead which I feel is better. Anyway, sorry for my rant.

My main problem with exposing this is that `cache.CacheDuration` stutters a bit. It should likely either be `cache.Duration` or `cache.CleanDuration`. I think the latter is better.

Reply:
Renamed it to `cache.ValidDuration` since it's really a measure of how long a cache entry stays valid after last access